iac-extensions/arm/src/test/java/org/sonar/iac/arm/tree/impl/bicep/IfConditionImplTest.java

```diff
@@ -26,44 +26,44 @@
 import org.sonar.iac.arm.parser.bicep.BicepLexicalGrammar;
 import org.sonar.iac.arm.tree.api.ArmTree;
 import org.sonar.iac.arm.tree.api.StringLiteral;
-import org.sonar.iac.arm.tree.api.bicep.IfExpression;
+import org.sonar.iac.arm.tree.api.bicep.IfCondition;
 
 import static org.sonar.iac.arm.ArmTestUtils.recursiveTransformationOfTreeChildrenToStrings;
 import static org.sonar.iac.common.testing.IacTestUtils.code;
 
-class IfExpressionImplTest extends BicepTreeModelTest {
+class IfConditionImplTest extends BicepTreeModelTest {
 
-  BicepParser parser = BicepParser.create(BicepLexicalGrammar.IF_EXPRESSION);
+  BicepParser parser = BicepParser.create(BicepLexicalGrammar.IF_CONDITION);
```

Comment:
Nitpick. The class already extends BicepTreeModelTest, but still the `parser` is used. Maybe it can be refactored?